### PR TITLE
docstrings: Fixing some incorrect docstrings.

### PR DESCRIPTION
**Pull request by simar7**
Signed-off-by: Simarpreet Singh s244sing@uwaterloo.ca

### CODE REVIEW

**Comment by simar7** [comment]
There are some others as well which mostly include a lack of period (".") at the end. Do we want to fix them as well? Although, grammatically they seem to be correct.

File: mozmill_automation/testrun.py [hunk 1]
```diff
@@ -438,7 +438,7 @@ def get_download_url(self):
             raise errors.NotFoundException('Could not read URL settings', filename)
 
     def run_tests(self):
-        """ Execute the normal and restart tests in sequence. """
+        """ Execute the remote tests and restart tests in sequence. """
```
Comment (whimboo) [inline mozmill_automation/testrun.py]: This line is still wrong. We are in the addon tests here.

**Comment by whimboo** [comment]
> There are some others as well which mostly include a lack of period (".") at the end. Do we want to fix them as well? Although, grammatically they seem to be correct.

If you want you can fix them for sure! I'm happy to review it. Thank you for the patch btw.

**Comment by whimboo** [comment]
@simar7 can you fix the remaining issue so that we can get your patch merged? Thanks.

**Comment by whimboo** [comment]
Btw. this issue will fix issue #35.

**Comment by whimboo** [comment]
We are going to discontinue Mozmill in favor of [Firefox UI Tests](https://github.com/mozilla/firefox-ui-tests). So I'm going to wont fix this PR.

**Pull request closed without merging**